sdk/database/helper/connutil/sql.go

```diff
@@ -53,8 +54,8 @@ type SQLConnectionProducer struct {
 	MaxIdleConnections       int         `json:"max_idle_connections" mapstructure:"max_idle_connections" structs:"max_idle_connections"`
 	MaxConnectionLifetimeRaw interface{} `json:"max_connection_lifetime" mapstructure:"max_connection_lifetime" structs:"max_connection_lifetime"`
 	DisableEscaping          bool        `json:"disable_escaping" mapstructure:"disable_escaping" structs:"disable_escaping"`
-	usePrivateIP             bool        `json:"use_private_ip" mapstructure:"use_private_ip" structs:"use_private_ip"`
 	SelfManaged              bool        `json:"self_managed" mapstructure:"self_managed" structs:"self_managed"`
+	usePrivateIP             bool
```

Comment:
Why are we removing the annotation here?

Reply:
The field is unexported so I think the struct tags can't actually do anything on it.